api/src/main/java/org/arquillian/cube/q/api/NetworkChaos.java

```diff
@@ -0,0 +1,138 @@
+package org.arquillian.cube.q.api;
```

Comment:
I thought the same but since originally it was done in this way, but I agree that the `Action` interface could be in its own file. Let's do it.